src/server.ts

```diff
@@ -33,27 +33,39 @@ class Honeybadger extends Client {
     }
   ]
 
-  public errorHandler: typeof errorHandler;
-  public requestHandler: typeof requestHandler;
-  public lambdaHandler: typeof lambdaHandler;
+  public errorHandler: typeof errorHandler
+  public requestHandler: typeof requestHandler
+  public lambdaHandler: typeof lambdaHandler
 
-  constructor(opts: Partial<Config> = {}) {
+  config: Config | ServerlessConfig
+
+  constructor(opts: Partial<Config | ServerlessConfig> = {}) {
     super({
       afterUncaught: fatallyLogAndExit,
       projectRoot: process.cwd(),
       hostname: os.hostname(),
       ...opts,
     })
+
+    // serverless defaults
+    const config = this.config as ServerlessConfig
+    config.reportTimeoutWarning = config.reportTimeoutWarning || true
+    config.timeoutWarningThresholdMs = config.timeoutWarningThresholdMs || 50
```

Comment:
This too. Is `0` an acceptable value? If so, this should be `??`.

Reply:
`0` should not be acceptable here, as it would probably fail to report to Honeybadger. I think `||` is OK here.